Source/carthage/Extensions.swift

```diff
@@ -57,19 +58,28 @@ extension GitURL: ArgumentType {
 
 /// Logs project events put into the sink.
 internal struct ProjectEventSink: SinkType {
+	let colorOptions: ColorOptions
+	
+	init(_ colorOptions: ColorOptions) {
```

Comment:
Normally, I don't give names to single-parameter initializers, but this has the potential to be confusing. For example, if you saw `ProjectEventSink(x)`, would you know what that meant?